daml-foundations/daml-ghc/base-rst-prefix.rst

```diff
@@ -3,8 +3,18 @@
 
 .. _stdlib-reference-base:
 
-Base for DAML 1.2
-=================
+The standard library
+======================
```

Comment:
```suggestion
====================
```
This will fail otherwise, won't it?

Reply:
no, it only complains if there's too much. but I can't remember why there was extra, removing it makes sense